server/server.go

```diff
@@ -905,7 +885,7 @@ func (s *Server) unversionedPost(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Server) v0DataPost(w http.ResponseWriter, r *http.Request) {
-	s.v0QueryPath(w, r, mux.Vars(r)["path"], false)
+	s.v0QueryPath(w, r, strings.TrimLeft(router.GetParams(r.Context())["path"], "/"), false)
```

Comment:
I think you actually want strings.TrimPrefix. strings.TrimLeft is a weird function, but in this case the result is the same, with a sole slash as input.

Reply:
`strings.TrimPrefix` is more unambiguous in its intent and functionality, so I agree.